odl/deform/linearized.py

```diff
@@ -136,13 +136,20 @@ class LinDeformFixedTempl(Operator):
     i.e., :math:`W_I'(v)^*(J)(x) = J(x) \, \\nabla I(x + v(x))`.
     """
 
-    def __init__(self, template):
+    def __init__(self, template, domain=None):
         """Initialize a new instance.
 
         Parameters
         ----------
         template : `DiscreteLpElement`
             Fixed template that is to be deformed.
+        domain : power space of `DiscreteLp`
+            The space of all allowable coordinates in the deformation.
+            A `ProductSpace` of ``template.ndim`` copies of a function-space.
+            It must fulfill
+            ``domain[0].partition == template.space.partition``, so
+            this option is useful mainly when different interpolations
+            in the displacement and template.
```

Comment:
It is not relevant here, giving a complex template is handled properly already without it.